copydb/config.go

```diff
@@ -70,6 +70,10 @@ type Config struct {
 
 	// The duration to wait for the replication to catchup before aborting. Only use if RunFerryFromReplica is true.
 	WaitForReplicationTimeout string
+
+	// Ghostferry will by default create tables on your target and fail if these already exist.
+	// This allows pre-existing tables on your target, schemas won't be validated.
```

Comment:
```suggestion
	// This allows pre-existing tables on your target, schemas and data compatibility won't be validated.
	// Use at your own risk
```

I feel like this should have a harder disclaimer 😄